dash_core_components/Store.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, id=Component.REQUIRED, storage_type=Component.UNDEFINED, data
         _locals.update(kwargs)  # For wildcard attrs
         args = {k: _locals[k] for k in _explicit_args if k != 'children'}
 
-        for k in ['id']:
+        for k in [u'id']:
```

Comment:
why did you do this?

Reply:
You commenting on auto generated code, I didn't do that.

Reply:
Ah ok.